src/main/java/UseCases/UseCaseInteractor.java

```diff
@@ -0,0 +1,250 @@
+package UseCases;
```

Comment:
Are you pulling from main? Some of these files don't look like they should be here. If possible can you rebase these commits so we get just your changes?